core/src/it/scala/org/ensime/core/JavaCompilerSpec.scala

```diff
@@ -113,7 +113,7 @@ class JavaCompilerSpec extends EnsimeSpec
           case "3" =>
             info.name shouldBe "java.io.PrintStream.println(java.lang.Object)"
             info.localName shouldBe "println"
-            info.`type`.name shouldBe "(java.lang.Object)void"
+            info.`type`.name shouldBe "public void println(Object arg0)"
```

Comment:
Btw, `JavaCompiler` will need a refactor soon, its getting nasty in there, perhaps somewhere around  #1366?